src/stepper/stepper.scss

```diff
@@ -29,8 +29,8 @@
     align-items: center;
     border-radius: 50%;
     display: flex;
-    width: $iui-l;
-    height: $iui-l;
+    width: $iui-l + $iui-xxs; // `+ $iui-xxs` accounts for border width
```

Comment:
again.. same.. maybe keep previous size? All these new ones kinda go against the ones we have set as standard.